mathics/builtin/arithmetic.py

```diff
@@ -2003,14 +1999,33 @@ def to_sympy(self, expr, **kwargs):
             if evaluation:
                 # Min and max might be Mathics expressions. If so, evaluate them.
                 for i in (1, 2):
-                    expr = var_min_max[i]
+                    min_max_expr = var_min_max[i]
                     if not isinstance(expr, Symbol):
-                        expr = expr.evaluate(evaluation)
-                        value = expr.to_sympy(**kwargs)
+                        min_max_expr_eval = min_max_expr.evaluate(evaluation)
+                        value = min_max_expr_eval.to_sympy(**kwargs)
```

Comment:
apart from the change in the name of the variables, is here something else?

Reply:
At one oint the mpmath and the sympy math were in the same function which made it _necessary_ to change the names; I needed the expr before evaluation and the min_max_expr before evaluation.

After they were split, in theory this didn't need to change. However, I think it a bad practice, which I hope we can stop doing ,to take  a variable of one type, e.g. Expression and convert it to another type e.g. SympyExpression as was happening in the function. I am not sure it was happening here, but it might have been.